minecraft_servers/sortygames/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "sortygames",
+  "nice_name": "Sorty - Games Netzwerk",
+  "direct_ip": "sortygames.net",
+  "supported_languages": [
+     "en",
+     "de"
+  ],
+  "social": {
+    "web": "https://www.sortygames.net",
+    "web_shop": "https://www.sortygames.net/ranks",
+    "web_support": "https://www.sortygames.net/discord",
+    "twitter": "sortygames",
+    "tiktok": "sortygames",
+    "instagram": "sortygames",
+    "discord": "https://www.sortygames.net/discord",
```

Comment:
Please use a discord invite url:
https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

Custom urls are only available for partner servers atm.